### PR TITLE
documentation cleanup

### PR DESCRIPTION
**Pull request by gyorb**
formatting and indentation fixes for markdown

### CODE REVIEW

File: docs/user_guide.md [hunk 1]
```diff
@@ -353,15 +352,15 @@ You can change the compilers that should be logged.
...
-~~~~~~~~~~~~~~~~~~~~~
+```sh
 export CC_LOGGER_GCC_LIKE="gcc:g++:clang"
-~~~~~~~~~~~~~~~~~~~~~
+```
 
 Example:
 
-~~~~~~~~~~~~~~~~~~~~~
+```
```
Comment (Xazax-hun) [inline docs/user_guide.md]: sh language is missing here

File: docs/user_guide.md [hunk 2]
```diff
@@ -401,14 +400,14 @@ viewed in a browser.
...
+```sh
 CodeChecker analyze ../codechecker_myProject_build.log -o my_plists
-~~~~~~~~~~~~~~~~~~~~~
+```
 
 `CodeChecker analyze` supports a myriad of fine-tuning arguments, explained
 below:
 
-~~~~~~~~~~~~~~~~~~~~~
+```sh
```
Comment (Xazax-hun) [inline docs/user_guide.md]: Do we want the sh here?

File: docs/user_guide.md [hunk 3]
```diff
@@ -1608,54 +1494,54 @@ a path glob pattern:
  * `-` results from the matching file paths will not be listed
 
 Example:
-~~~~~~~~~~~~~~~~~~~~~
+```sh
```
Comment (Xazax-hun) [inline docs/user_guide.md]: Do we want sh syntax highlighting for these files?

File: docs/user_guide.md [hunk 4]
```diff
@@ -1608,54 +1494,54 @@ a path glob pattern:
...
+```
 `x.cpp` will be included in the run results and all other files under `/a/b/`
 path will not be included.
 
-#### <a name="list-source-components"></a> List source components
+#### List source components <a name="list-source-components"></a>
 List the name and basic information about source component added to the
 server.
-~~~~~~~~~~~~~~~~~~~~~
+```sh
```
Comment (Xazax-hun) [inline docs/user_guide.md]: Do we want sh syntax highlight for helps?

**Comment by csordasmarton** [comment]
@gyorb Please resolve the merge conflict and check Xazax comments.

**Pull request merged**